app/views/users/show.html.erb

```diff
@@ -165,6 +165,12 @@
 
     <div class="user-description richtext text-break"><%= @user.description.to_html %></div>
 
+    <% if current_user and @user.id == current_user.id %>
+      <div class="my-3">
+        <%= link_to t(".edit_profile"), edit_profile_path, :class => "btn btn-outline-primary" %>
```

Comment:
I'm not sure about the placement of this - it looks a bit odd to me. That may be partly because it's above the extra user data line that administrators like me get but I think it would stick out a bit without that.

Could we maybe put it on the right at the top? or would that be an issue on small screens?

Reply:
I can move it around, but to be fair I think the whole page is already a mess. My plan is a followup PR that 

* Moves the profile content out of the (grey) page header and into the main part of the page
* Remove the strange override that makes the profile page the only edge-to-edge page
* Moves the admin stuff into the header along with the rest of the stuff that's there already
* Moves the non-profile stuff (like lists of friends, links that aren't shown to other people etc) off that page and into a separate dashboard

I think at that point the edit button will sit more naturally under the profile content.

Reply:
Sounds reasonable to me...